src/app/components/SearchResults/SearchHeader.jsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Select from '../Form/Select';
+import { initialSearchQuery, searchQueryPropTypes } from '../../stores/InitialState';
+import { getQueryString } from '../../search/query';
+
+const SearchHeader = ({
+  metadata, searchQuery, userQuery, router,
+}) => {
+  // page for query is -1 page shown
+  const totalPages = Math.floor((Number(metadata.total) - 1) / Number(searchQuery.per_page || 10)) + 1;
+  // return list of pages till total pages
+  const pageList = [];
+  for (let i = 1; i <= totalPages; i += 1) {
+    pageList.push(i);
+  }
+
+  // redirect to url with query params
+  const submit = (query) => {
+    const path = `/search?${getQueryString(query)}`;
+    router.push(path);
+  };
+
+  // update page in store and go to any page
+  const goToPage = (pageNumber) => {
+    const newPage = Number(pageNumber) - 1;
+    const perPage = searchQuery.per_page;
+    const newQuery = Object.assign({}, searchQuery, { page: newPage, per_page: perPage, total: metadata.total });
+    userQuery(newQuery);
+    submit(newQuery);
+  };
+
+  // click and navigate to any page
+  const navigateToPage = (e, pageNumber) => {
+    e.preventDefault();
+    e.stopPropagation();
+    let page = pageNumber;
+    if (page < 1) {
+      page = 1;
+    }
+    if (page > totalPages) {
+      page = totalPages;
+    }
+    goToPage(page);
+  };
+
+  // update page in store when select of pages change
+  const onChangePage = (e) => {
+    goToPage(Number(e.target.value));
+  };
+
+  // update per_page in store when select of per_ppage changes
+  const onChangePerPage = (e) => {
+    const newPage = 0;
+    const newPerPage = e.target.value;
+    if (newPerPage !== searchQuery.per_page) {
+      const newQuery = Object.assign({}, searchQuery, { page: newPage, per_page: newPerPage, total: metadata.total });
+      userQuery(newQuery);
+      submit(newQuery);
+    }
+  };
+
+  return (
```

Comment:
The search header returns when there are no results which doesn't seem like an ideal output. Do we have designs for a no results page? Also, the pagination links show as active links when there is only 1 page. These should be disabled or removed when only one page of results is returned.

Reply:
In the design, the search header shows when there is no results, and the pagination is still active. I will change the message on the number of items. We need to add a message in main results when there is no results according to the design, not sure if it's better to make it in other ticket

Reply:
Leave it as-is now and we'll add a new ticket for this since there may be some thought from Jack on this.

Reply:
I just added the empty results msg as in the design. Btw we should check the fonts